code/modules/mob/living/carbon/brain/MMI.dm

```diff
@@ -22,6 +22,9 @@
 	var/obj/mecha = null//This does not appear to be used outside of reference in mecha.dm.
 
 	attackby(var/obj/item/O as obj, var/mob/user as mob, params)

```

Comment:
Relative pathing was already there, I'm under no obligation to actually fix every line of the file.